packages/pwa-kit-create-app/assets/bootstrap/js/overrides/app/utils/translations.js

```diff
@@ -0,0 +1,34 @@
+/*
```

Comment:
Hmm, strangely I did not see this file in the generated project. 

I ran the following command: `node packages/pwa-kit-create-app/scripts/create-mobify-app-dev.js --outputDir generated-project` and then chose the demo instance when prompted.

Reply:
Oh nvm, I had to clear the npx cache first by: `rm -rf ~/.npm/_npx`. This should be done automatically by `create-mobify-app-dev`.

All right, now I can see the file in the generated project.

Reply:
local npm cache is probably stuck

Reply:
@vmarta That's strange! You should be able to see the file under `overrides/app/utils`

I intermedeately ran into a heavy caching problem where the generated app kept giving me older files that I had already deleted. It kept copying old files which did not even exist in the monorepo. I ended up temporarily bumping version to something random to have the generator pick the latest files.